sdcm/cluster_aws.py

```diff
@@ -702,6 +702,40 @@ def image(self):
     def ena_support(self) -> bool:
         return self._instance.ena_support
 
+    def create_and_attach_new_volume(self, volume_name_char, availability_zone, size, volume_type, tags=None,
```

Comment:
Maybe it would be clearer if '/dev/xvf' would be the input
also I don't understand why in some part of the API you refer it as `/dev/sdf`, and on the machine as `/dev/xvf` ?

I think the user of this function should know only one of them, (i prefer the one we actually use on the machine)